src/simpleitk_timeseries_motion_correction/motion.py

```
@@ -150,9 +151,49 @@ def voxelwise_std(image_list):
     return std_image
 
 
+def estimate_shrinks_sigmas(img, level=2):
+    min_length = min(img.GetSize()[:3])
+    min_spacing = min(img.GetSpacing()[:3])
+    
+    shrinks_l_l, smooths_l_l, iterations_l_l = make_syn_pyramid(
```

Ruff (0.14.14)

[warning] Unpacked variable `iterations_l_l` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)